libs/cardano-protocol-tpraos/test/Test/Cardano/Protocol/Binary/CddlSpec.hs

```diff
@@ -27,20 +28,25 @@ spec :: Spec
 spec =
   describe "CDDL" $ do
     let n = 3
-    specForEra @ShelleyEra readShelleyCddlFiles n
+    specForEra @ShelleyEra readShelleyCddlFiles shelleyCDDL n
 
-specForEra :: forall era. Era era => IO [BSL.ByteString] -> Int -> Spec
-specForEra readCddlFiles n = do
+specForEra :: forall era. Era era => IO [BSL.ByteString] -> Huddle -> Int -> Spec
+specForEra readCddlFiles cddlFile n = do
   describe (eraName @era) $ do
     let v = eraProtVerLow @era
-    beforeAllCddlFile n readCddlFiles $ do
-      cddlRoundTripAnnCborSpec @(BHeader StandardCrypto) v "header"
-      cddlRoundTripCborSpec @(BHeader StandardCrypto) v "header"
-      cddlRoundTripCborSpec @(BHBody StandardCrypto) v "header_body"
-      cddlRoundTripCborSpec @(CBORGroup (OCert StandardCrypto)) v "[ operational_cert ]"
-      -- TODO: add Huddle round trip tests
-      describe "DecCBOR instances equivalence via CDDL" $ do
-        cddlDecoderEquivalenceSpec @(BHeader StandardCrypto) v "header"
-    describe "DecCBOR instances equivalence via CDDL - Huddle" $ do
-      specWithHuddle shelleyCDDL 100 $ do
+    describe "Ruby-based" $
+      beforeAllCddlFile n readCddlFiles $ do
+        cddlRoundTripAnnCborSpec @(BHeader StandardCrypto) v "header"
+        cddlRoundTripCborSpec @(BHeader StandardCrypto) v "header"
+        cddlRoundTripCborSpec @(BHBody StandardCrypto) v "header_body"
+        cddlRoundTripCborSpec @(CBORGroup (OCert StandardCrypto)) v "[ operational_cert ]"
+        describe "DecCBOR instances equivalence via CDDL" $ do
+          cddlDecoderEquivalenceSpec @(BHeader StandardCrypto) v "header"
+    describe "Huddle" $ specWithHuddle cddlFile 100 $ do
+      huddleRoundTripAnnCborSpec @(BHeader StandardCrypto) v "header"
+      huddleRoundTripCborSpec @(BHeader StandardCrypto) v "header"
+      huddleRoundTripCborSpec @(BHBody StandardCrypto) v "header_body"
+      xdescribe "[ operational_cert ] doesn't roundtrip with huddle" $
```

Comment:
Is this a known issue? It does ring a bell, but couldn't find a github issue. If there isn't one, I will create one.

Reply:
There was a relevant ticket, but I thought it was fixed: https://github.com/IntersectMBO/cardano-ledger/issues/3559

Feel free to create a new one and figure it out what is going on with it.